src/auth/src/credentials/idtoken/mds.rs

```diff
@@ -161,11 +161,13 @@ impl Builder {
     ///
     /// ```
     /// # use google_cloud_auth::credentials::idtoken;
+    /// # tokio_test::block_on(async {
```

Comment:
Why the async block if there are no `.await` calls? And do you really need to compile and **run** these examples? Or would it be enough to put them in an asynchronous function so they are compiled?

Reply:
removed some of the async blocks and replaced with `no_run` and simple functions when needed